src/components/timeline/TrackCounterGraph.js

```diff
@@ -172,6 +205,11 @@ type OwnProps = {|
   +counterIndex: CounterIndex,
   +lineWidth: CssPixels,
   +graphHeight: CssPixels,
+  +graphType: GraphType,
+  +graphStrokeColor: CanvasPattern,
+  +graphFillColor: string,
+  +graphDotColor: CanvasPattern,
+  +renderTooltip: any,
```

Comment:
This needs a type, the `any` type means "opt out of type checking" which is dangerous. I believe the type is:

`(counterIndex: number) => React.Node`